toolchain/lowering/lowering_function_context.h

```diff
@@ -48,13 +66,25 @@ class LoweringFunctionContext {
     return lowering_context_->GetType(type_id);
   }
 
+  // Create a synthetic block that corresponds to no SemanticsNodeBlockId. Such
+  // a block should only ever have a single predecessor, and is used when we
+  // need multiple `llvm::BasicBlock`s to model the linear control flow in a
+  // single SemanticsIR block.
+  auto CreateSyntheticBlock() -> llvm::BasicBlock*;
+
+  // Determine whether block_ is the most-recently-created synthetic block.
```

Comment:
```suggestion
  // Determine whether block is the most-recently-created synthetic block.
```

Oops, missed this, sorry.